crates/typst-cli/src/args.rs

```diff
@@ -132,8 +133,51 @@ pub struct SharedArgs {
         value_parser = clap::value_parser!(DiagnosticFormat)
     )]
     pub diagnostic_format: DiagnosticFormat,
+
+    /// Add an input value visible through sys.inputs
```

Comment:
It should probably clarify that this will be a string value, i.e, something like `-i var=int(10)` will be `"int(10)"`, not `10`.

Reply:
I agree it should be noted in the documentation, but as noted by @laurmaedje a description too long doesn't do too well for the CLI docs.
I'd propose putting it in the more extensive documentation / usage guide.